shell/platform/tizen/flutter_tizen.cc

```diff
@@ -216,29 +164,40 @@ void FlutterRegisterViewFactory(
           view_type, std::move(view_factory)));
 }
 
+// Returns the texture registrar corresponding to the given opaque API handle.
+static FlutterTizenTextureRegistrar* TextureRegistrarFromHandle(
+    FlutterDesktopTextureRegistrarRef ref) {
+  return reinterpret_cast<FlutterTizenTextureRegistrar*>(ref);
+}
+
+// Returns the opaque API handle for the given texture registrar instance.
+static FlutterDesktopTextureRegistrarRef HandleForTextureRegistrar(
+    FlutterTizenTextureRegistrar* registrar) {
+  return reinterpret_cast<FlutterDesktopTextureRegistrarRef>(registrar);
+}
+
 FlutterDesktopTextureRegistrarRef FlutterDesktopRegistrarGetTextureRegistrar(
     FlutterDesktopPluginRegistrarRef registrar) {
-  FT_UNIMPLEMENTED();
-  return nullptr;
+  return HandleForTextureRegistrar(registrar->engine->texture_registrar());
```

Comment:
```texture_registrar_``` is initialized only when engine is ```headed```. Therefore, I think it would be good if you could add a warning log for headless

Reply:
If in headleass mode, it will print ad warning log.